lib/internal/fs/watchers.js

```diff
@@ -161,7 +162,9 @@ FSWatcher.prototype.start = function(filename,
     const error = errors.uvException({
       errno: err,
       syscall: 'watch',
-      path: filename
+      path: filename,
+      description: err === UV_ENOSPC ?
```

Comment:
Nit: Might want to call it message?

Reply:
+1 to call it `message`. By the way if the key is `message` it currently will not appear in the final error object as an additional property whereas `description` will.

Reply:
ping @addaleax